portal-knightsconfig.json

```diff
@@ -0,0 +1,438 @@
+        "FieldName":"saveFolderPath",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.basicServerData.saveFolderPath",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"./savedata"
+    },
+    {
+        "DisplayName":"Gameplay Mode",
+        "Category":"Server Settings",
+        "Description":"The game play mode for the dedicated server.",
+        "Keywords":"gameplay,Mode",
+        "FieldName":"gameplayMode",
+        "InputType":"enum",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.gameplayMode",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"Adventure",
+        "EnumValues":{
+            "Adventure": "Adventure",
+            "Creative": "Creative"
+        }
+    },
+    {
+        "DisplayName":"Universe Size",
+        "Category":"Server Settings",
+        "Description":"The universe size for the dedicated server.",
+        "Keywords":"universe,Size",
+        "FieldName":"universeSize",
+        "InputType":"enum",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.universeSize",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"Normal",
+        "EnumValues":{
+            "Small": "Normal",
+            "Normal": "Large",
+            "Large": "Huge"
+        }
+    },
+    {
+        "DisplayName":"Universe ID",
+        "Category":"Server Settings",
+        "Description":"The world slot used when accessing this dedicated server.",
+        "Keywords":"universe,Id",
+        "FieldName":"universeId",
+        "InputType":"number",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.universeId",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"0",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Hide Console Window",
+        "Category":"Server Settings",
+        "Description":"This will hide the console window and break the console in AMP if changed.",
+        "Keywords":"",
+        "FieldName":"hideConsoleWindow",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.hideConsoleWindow",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"false",
+        "EnumValues":{},
+        "Hidden": true
+    },
+    {
+        "DisplayName":"Admin Password",
+        "Category":"Server Settings",
+        "Description":"Choose the password used to access your dedicated server by the Admin user group.",
+        "Keywords":"admin,password",
+        "FieldName":"adminscredentialspassword",
+        "InputType":"RandomPassword",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.admins.credentials.password",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"admin_password_please_change",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Admin Can Login",
+        "Category":"Server Settings",
+        "Description":"Defines whether Admins can log in.",
+        "Keywords":"admin,can,Login",
```

Comment:
"can" is unnecessary as a search term? Same applies for all the relevant settings below